bedrock/products/views.py

```diff
@@ -140,17 +145,23 @@ def _filter_articles(articles_list, category):
 
 def resource_center_landing_view(request):
 
+    template_name = "products/vpn/resource-center/landing.html"
+
     ARTICLE_GROUP_SIZE = 6
 
-    active_locales = [
-        "en-US",  # Initially, en-US is the only one available in Contentful
-    ]
+    active_locales = get_active_contentful_locales(
+        classification=CONTENT_CLASSIFICATION_VPN,
+        content_type=CONTENT_TYPE_PAGE_RESOURCE_CENTER,
+    )
+
     locale = l10n_utils.get_locale(request)
 
-    template_name = "products/vpn/resource-center/landing.html"
-    ctx = {
-        "active_locales": active_locales,
-    }
+    # If you go to the VRC for a language we're working on but
+    # which isn't active yet because it doesn't meet the activation
+    # threshold percentage, we should send you to the default locale
+    if locale not in active_locales:
+        _path = reverse("products.vpn.resource-center.landing")
+        return redirect(f"/{DEFAULT_LOCALE}{_path}")
```

Comment:
you could redirect to the URL without a locale and let bedrock decide which is best for the user.

Reply:
I just started amending this but then reverted. I _think_ this went in as an explicit fallback to `en-US`/`DEFAULT_LOCALE` because if you reach here you are trying to use a locale which is not active, and if there's a possibility that if your browser is _insisting_ on a that locale, you'll just end up in a redirect loop seeking an unavailable locale.

I could be wrong - there may be a gap in my understanding of Bedrock's automatic locale selection that avoids the redirect loop problem, but if not, I think this is going to give us a usable fallback that's avoids the risk of that loop.

What d'ya think?

Reply:
Maybe we need to think this through a bit more. Bedrock will choose the best active locale, but that happens in the `l10n_utils.render` function, which is called after this view. So maybe the best option if we find this case is to just call render with the active locales and let it handle the redirect?

Reply:
Yeah that sounds a lot simpler. Will sort it 🎸